docs/dev/testing.rst

```diff
@@ -321,41 +322,25 @@ with the construction of that database. For example, the output routines:
 
 We also use this option to run the data validations.
 
-Assuming you do want to run the ETL and build new databases as part of the test
-you're running, the contents of that database are determined by an ETL settings
-file. By default, the settings file that's used is
-``test/settings/integration-test.yml`` But it's also possible to use a
-different input file, generating a different database, and then run some
-tests against that database.
-
-For example, we test that FERC 1 data can be loaded into a PUDL database all
-by itself by running the ETL tests with a settings file that includes only A
-couple of FERC 1 tables for a single year. This is the ``ferc1_solo`` Tox
-test environment:
-
-.. code-block:: console
-
-  $ pytest --etl-settings=test/settings/ferc1-solo-test.yml test/integration/etl_test.py
-
-Similarly, we use the ``test/settings/full-integration-test.yml`` settings file
-to specify an exhaustive collection of input data, and then we run a test that
-checks that the database schemas extracted from all historical FERC 1 databases
-are compatible with each other. This is the ``ferc1_schema`` test:
-
-.. code-block:: console
-
-  $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
-
-The raw input data that all the tests use is ultimately coming from our
-`archives on Zenodo <https://zenodo.org/communities/catalyst-cooperative>`__.
-However, you can optionally tell the tests to look in a different places for more
-rapidly accessible caches of that data and to force the download of a fresh
-copy (especially useful when you are testing the datastore functionality
-specifically). By default, the tests will use the datastore that's part of your
-local PUDL workspace.
-
-For example, to run the ETL portion of the integration tests and download
-fresh input data to a temporary datastore that's later deleted automatically:
+Assuming you do want to run the ETL and build new databases as part of the test you're
+running, the contents of that database are determined by an ETL settings file. By
+default, the settings file that's used is
+``src/pudl/package_data/settings/etl_fast.yml`` But it's also possible to use a
+different input file, generating a different database, and then run some tests against
+that database.
+
+We use the ``src/pudl/package_data/etl_full.yml`` settings file to specify an exhaustive
+collection of input data.
```

Comment:
Updated the paths to the settings files that live in the repo, as they were out of date.

Reply:
Yeah, it would be nice.  We have some dynamic docs building stuff for like, the data dictionaries, but this kind of thing is just doomed to get out of date!